lib/mayaUsd/fileio/jobs/jobArgs.h

```diff
@@ -240,33 +245,16 @@ struct UsdMayaJobExportArgs
     MAYAUSD_CORE_PUBLIC
     static const VtDictionary& GetDefaultDictionary();
 
-    /// Adds type name to filter out during export. This will also add all
-    /// inherited types (so if you exclude "constraint", it will also exclude
-    /// "parentConstraint")
-    MAYAUSD_CORE_PUBLIC
-    void AddFilteredTypeName(const MString& typeName);
```

Comment:
There were no clients modifying this variable mid-export. The list was always built as the arguments were parsed and left untouched for the duration of the export.